index.js

```diff
@@ -1,27 +1,73 @@
 var $ = require('jquery')
+var Mustache = require('mustache')
 
-function AnnotationPoller (loaded) {
+function AnnotationPoller (opts, loaded) {
   var _this = this
+  this.pollInterval = opts.pollInterval || 3000
   this.endpoint = '/api/v1/annotations.js'
   this.annotations = {}
+  this.template = '<li id="annotation-{{id}}" style="{{status}}"><span>{{description}}</span><a href="{{{external-link}}}">{{external-link-text}}</a></li>'
+  this.addonSelector = '#npm-addon-box'
 
   $(document).ready(function () {
-    _this.load(function () {
-      return loaded()
-    })
+    _this.start(loaded)
```

Comment:
Been a while since I've done frontend work, but why not let the consumer of this module define how/when the poller starts?